### PR TITLE
update apps for 2004

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8017

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/application-management/apps-in-windows-10.md [hunk 1]
```diff
@@ -39,53 +39,53 @@ You can list all provisioned Windows apps with this PowerShell command:
 Get-AppxProvisionedPackage -Online | Format-Table DisplayName, PackageName
 ```
 
-Here are the provisioned Windows apps in Windows 10 versions 1803, 1809, 1903, and 1909.
+Here are the provisioned Windows apps in Windows 10 versions 1803, 1809, 1903, 1909 and 2004.
```
Comment (JohanFreelancer9) [inline windows/application-management/apps-in-windows-10.md:42]:
```suggestion
Here are the provisioned Windows apps in Windows 10 versions 1803, 1809, 1903, 1909, and 2004.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - Please note: The table does not render correctly. Also, there is one minor edit.

**Comment by VLG17** [comment]
> Copy review - Please note: The table does not render correctly. Also, there is one minor edit.

@JohanFreelancer9 Could you please let me know what i need to change to have it render correctly?

File: windows/application-management/apps-in-windows-10.md [hunk 2]
```diff
@@ -39,53 +39,53 @@ You can list all provisioned Windows apps with this PowerShell command:
 Get-AppxProvisionedPackage -Online | Format-Table DisplayName, PackageName
 ```
 
-Here are the provisioned Windows apps in Windows 10 versions 1803, 1809, 1903, and 1909.
+Here are the provisioned Windows apps in Windows 10 versions 1803, 1809, 1903, 1909, and 2004.
 
-| Package name                                 | App name                                                                                                           | 1803 | 1809 | 1903 | 1909 | Uninstall through UI? |
+| Package name                                 | App name                                                                                                           | 1803 | 1809 | 1903 | 1909 | 2004 | Uninstall through UI? |
 |----------------------------------------------|--------------------------------------------------------------------------------------------------------------------|:----:|:----:|:----:|:----:|:---------------------:|
```
Comment (JohanFreelancer9) [inline windows/application-management/apps-in-windows-10.md:45]:
```suggestion
|----------------------------------------------|--------------------------------------------------------------------------------------------------------------------|:----:|:----:|:----:|:----:|:----:|:---------------------:|
```

Comment (JohanFreelancer9) [inline windows/application-management/apps-in-windows-10.md:45]: @VLG17  - Unfortunately I cannot check if it works, but I think this should do it.

Comment (JohanFreelancer9) [inline windows/application-management/apps-in-windows-10.md:45]:
```suggestion
|----------------------------------------------|--------------------------------------------------------------------------------------------------------------------|:----:|:----:|:----:|:----:|:----:|:---------------------:|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@VLG17 - Unfortunately I cannot check if it works, but I think this should do it.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**